src/components/CustomerQuota/ItemsSelection/ItemContent.tsx

```diff
@@ -60,6 +62,13 @@ export const ItemContent: FunctionComponent<{
           <ItemMaxUnitLabel unit={unit} maxQuantity={maxQuantity} />
         </AppText>
       )}
+      {descriptionAlert && descriptionAlert.length > 0 && (
+        <ShowAddonsToggle
+          descriptionAlert={descriptionAlert}
+          toggleIsShowAddons={showAddonsToggle}
+          isShowAddons={showAddons}
+        />
```

Comment:
We should support the following scenario:
If there's no `showAddonsToggle`, show the text only and not render `ShowAddonsToggle`